src/main/resources/reference.conf

```diff
@@ -0,0 +1,13 @@
+latis {
+  hapi {
+    about {
+      # Environment variable overrides
```

Comment:
Is this because they don't automatically look for env vars by munging names like LatisProperties does or is there some other precedent issue here?

Reply:
@lindholc ?

Reply:
I'm not sure.

Reply:
They don't automatically look for environment variables except for substitutions (from https://github.com/lightbend/config#features-of-hocon):

> - substitutions fall back to environment variables if they don't resolve in the config itself, so `${HOME}` would work as you expect.
> - substitutions normally cause an error if unresolved, but there is a syntax `${?a.b}` to permit them to be missing.